chromeos.kernelci.org

```diff
@@ -110,6 +110,30 @@ cmd_docker() {
         cros-tast \
         cros-sdk
     cd -
+
+    echo "Updating Docker images"
+    # Build the images with kci_docker
+    cd checkout/kernelci-core/config/docker
+    core_rev=$(git show --pretty=format:%H -s origin/chromeos.kernelci.org)
+    rev_arg="--build-arg core_rev=$core_rev"
+    px_arg='--prefix=kernelci/staging-'
```

Comment:
This will overwrite the staging images.  Instead, the prefix should be `kernelci/cros-`.

Reply:
It will not, because i use fragment cros-kernelci for all images, so tag name will be different
https://github.com/kernelci/kernelci-core/pull/1450/commits/8f7d7e7e402450dd9a2e7e8a3b2c00caf15ce5e9